irohad/execution/impl/common_executor.cpp

```diff
@@ -16,6 +16,7 @@
  */
 
 #include <algorithm>
+
 #include "common/types.hpp"
 #include "execution/common_executor.hpp"
```

Comment:
move it into beginning
Please, check https://google.github.io/styleguide/cppguide.html#Names_and_Order_of_Includes